hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -109,25 +105,36 @@ public SignedStateBalancesExporter(
 		this.signer = signer;
 		this.expectedFloat = properties.getLongProperty("ledger.totalTinyBarFloat");
 		this.dynamicProperties = dynamicProperties;
+		exportPeriod = dynamicProperties.balancesExportPeriodSecs();
+	}
+
+	private Instant getFirstExportTime(Instant now, final int exportPeriodInSecs) {
+		final long epochSeconds = now.getEpochSecond();
+		long remainingSecs = epochSeconds % exportPeriodInSecs;
+		if(remainingSecs == 0) {
+			return Instant.ofEpochSecond(now.getEpochSecond());
+		}
+		else {
```

Comment:
nit `else` not needed.

Reply:
Yeah, removed. The original intension is to make these two different cases more obvious.